assets/css/style.css

```diff
@@ -70,10 +70,18 @@ input#height {
 	width: 3em;
 }
 
+[mv-multiple="resolution"] {
+	margin: 0 2px 0 2px;
+}
```

Comment:
If you use pixels here, that means if the font size ever changes, you need to adjust your spacing. Remember DRY? It applies to CSS as well. In some cases repetition is unavoidable, but in this case, we can actually express our intent better with code, by using `em` units, which scale nicely with the font size. Always use `em`s if something would benefit by scaling with the text.